packages/runtime-core/src/componentProxy.ts

```diff
@@ -42,28 +54,12 @@ export const PublicInstanceProxyHandlers = {
       // return the value from propsProxy for ref unwrapping and readonly
       return propsProxy![key]
     } else {
-      // TODO simplify this?
+      if (hasOwn(proxyHandlers, key)) {
+        return target[proxyHandlers[key]]
+      }
       switch (key) {
-        case '$data':
-          return data
-        case '$props':
-          return propsProxy
```

Comment:
`data` and `propsProxy` not include in `target`

Reply:
they both were taken from it earlier
![image](https://user-images.githubusercontent.com/19504461/66668708-7d9c7f80-ec5e-11e9-8c88-3af74c6171a1.png)

Reply:
Yep, sorry

Reply:
no problems